src/components/sidebar/routesView/RouteShow.tsx

```diff
@@ -66,9 +66,22 @@ class RouteShow extends React.Component<IRouteShowProps> {
         );
     }
 
-    private renderRoutePaths = () => {
-        const routePaths = this.props.route.routePaths;
+    private groupRoutePathsOnDates = (routePaths: IRoutePath[]) => {
+        const res = {};
+        routePaths.forEach((rp) => {
+            // tslint:disable-next-line:max-line-length
+            const identifier = `${rp.startTime.toLocaleDateString()}-${rp.endTime.toLocaleDateString()}`;
```

Comment:
Could simplify to `rp.startTime.toLocaleDateString() + rp.endTime.toLocaleDateString()` for this purpose?

Reply:
sure